Documentation/Books/Manual/Security/ChangeRootPassword.md

```diff
@@ -0,0 +1,32 @@
+# Howto Reset Root Password
+
+One can reset Root Password in the following way:
+
+1) Stop the server
+2) Set `authentication=false` in the `arangod.conf` file
```

Comment:
in the / from / from the arangod.file used - i would use only "in the arangod.conf file"